### ISSUE TITLE
Trouble running

### ISSUE THREAD

**Issue by SimonDanisch** [GPUOpen-LibrariesAndSDKs/RadeonProRender-Baikal#125]
First I needed to copy `msvcr110.dll` && `msvcp110.dll` from `C:\Program Files (x86)\Windows Kits\10\App Certification Kit` into the folder, even though I have the redistributional kit installed correctly.

Unrelated question: Wasn't there an option to cycle through differenct scenes in the stand alone? I thought I was able to test more complex scenes at some point...

**Comment by yozhijk** [comment]
This is probably a dependency caused by OpenImageIO, we will update it shortly. We have also separated image loader into a separate library recently, so Baikal core does not depend on it anymore. 

As regards scenes, you can load whatever obj scene you want using -p path -f file command line options.

**Comment by SimonDanisch** [comment]
Thanks for the quick reply!
I guess my problem is that the test scenes don't seem to get shipped anymore?
I see in https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRender-Baikal/blob/2308d552ad0993d93c315c8d94862c1c146b774d/BaikalTest/test_scenes.h#L138
e.g. ` "../Resources/TestData/Classroom/classroom.obj"` but I can't find that file anywhere...
After some more searching, they seem to be hosted in https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRender-Tests. Should I file a PR to add that to the description?

**Comment by SimonDanisch** [comment]
Although, when I download those, it doesn't really seem to work and the app just stops working:
```
PS C:\Users\sdani\gpustuff\RadeonProRender-Baikal\BaikalStandalone> ..\build\bin\Release\BaikalStandalone.exe -platform
1 -f C:\Users\sdani\gpustuff\RadeonProRender-Baikal\BaikalStandalone\Rungholt\rungholt.obj
Running on devices:
0: GeForce GTX 1060
OpenGL interop mode enabled
[stops working]
```

**Comment by yozhijk** [comment]
-p should contain the path to the model and -f just a filename, so if you downloaded the model to Resources/classroom, simply run it as BaikalStandalone64.exe -p ../Resources/classroom -f classroom.obj

It uses -p directory as a base directory for resource loading

**Comment by yozhijk** [comment]
Would be great if you can add a note on scenes locations, sure

**Comment by SimonDanisch** [comment]
Ok that actually works ;) Any chance to have an error, instead of just quitting the program? :) 
I noticed, that the defaults are not very great:
![grafik](https://user-images.githubusercontent.com/1010467/38412210-210bf324-3989-11e8-80e9-a8e34ffc05ea.png)
I think last time I checked, everything was much brighter.
Anyways, maybe I should just download the blender plugin to get a better idea about complex scene performance + looks.
Thanks for the quick responses!
